modules/dataset.py

```diff
@@ -12,41 +13,49 @@
 class MultDataset(Dataset):
     def __init__(self, split):
         self.split = split
-        
+        self.data = dict()
+
+        #Open Audio Features
         with open('audio_features_' + self.split + '.sav') as f:
             audio = joblib.load(f)
 
+        #Open Video Features
         with open('video_features_' + self.split + '.sav') as f:
             video = joblib.load(f)
 
+        #Open Text Features
         with open('text_features_' + self.split + '.sav') as f:
             text = joblib.load(f)
 
-        self.data = dict()
+        #Open File According to Split
+        with open('./data/' + self.split + '_split.csv', 'r') as file:
+            csv_reader = csv.reader(file)
+            next(csv_reader)
 
-        with open('../data/train_split.csv', 'r') as csv_file:
```

Comment:
@sreekanth-sreekumar - it seemed that this file was never used after opening - any reason for it?

Reply:
Yeah, it needs to be used once we figure out masking. It's a placeholder for now